src/main/java/frc/robot/Constants.java

```diff
@@ -148,4 +150,54 @@ public static final class AutoConstants {
     public static final double kMaxAngularSpeedRadiansPerSecond = Math.PI;
     public static final double kMaxAngularSpeedRadiansPerSecondSquared = Math.PI;
   }
+
+  public static class FieldConstants {
+    public static final double kFieldLength = 4.0;
+    public static final double kFieldWidth = 3.0;
+  }
+
+  public static class PhotonVisionConstants {
+    public static final String kCameraName1 = "Camera1";
+    public static final String kCameraName2 = "Camera2";
+    public static final Transform3d kCamera1ToRobotOffset = new Transform3d();
+    public static final Transform3d kCamera2ToRobotOffset = new Transform3d(0.0, -0.12, 0.0, new Rotation3d());
+
+
+    public static final double kCamera1HeightMeters = 0.215;
+    public static final double kCamera2HeightMeters = 0.0;
+    public static final double kCamera3HeightMeters = 0.0;
+
+    public static final double KCameraPitchRadians = 0.0;
+
+    public static final double kTarget1HeightMeters = 0.540;
+    public static final double kTarget2HeightMeters = 0.522;
+    public static final double kTarget3HeightMeters = 0.530;
+    public static final double kTarget4HeightMeters = 0.0;
+    public static final double kTarget5HeightMeters = 0.0;
+    public static final double kTarget6HeightMeters = 0.0;
+    public static final double kTarget7HeightMeters = 0.0;
+    public static final double kTarget8HeightMeters = 0.0;
```

Comment:
A later diff removes uses of these constants, but the constants appear to remain, no longer used.